todos/todos.js

```diff
@@ -17,28 +17,51 @@ const deleteButton = document.querySelector('.delete-button');
 
 // let some todo state (an array)
 
+let todoArray = [];
+
 todoForm.addEventListener('submit', async (e) => {
     // on submit, 
+    e.preventDefault();
+    const data = new FormData(todoForm);
     // create a todo in supabase using for data
+    const todo = data.get('todo');
+    await createTodo(todo);
     // reset the form DOM element
+    
```

Comment:
the form input isn't clearing on 'submit', this can be fixed by resetting the form with -- todoForm.reset()